<ui version="4.0" >
 <author>Andres Cabrera</author>
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QuteCsound Configuration</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>460</y>
     <width>621</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="editorTabWidget" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Editor</string>
    </attribute>
    <widget class="QFontComboBox" name="fontComboBox" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>20</y>
       <width>195</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Font</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Size</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fontSizeComboBox" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>50</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <item>
      <property name="text" >
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>36</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>48</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>80</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>Autoplay when launched from file</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Run</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>100</y>
       <width>591</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title" >
      <string>File offline</string>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>110</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QCheckBox" name="InputFilenameCheckBox" >
         <property name="text" >
          <string>Input Filename</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="InputFilenameLineEdit" />
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="toolButton_9" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="OutputFilenameCheckBox" >
         <property name="text" >
          <string>Output Filename</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="OutputFilenameLineEdit" />
       </item>
       <item row="1" column="2" >
        <widget class="QToolButton" name="toolButton_10" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>30</y>
        <width>172</width>
        <height>72</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="FileOverrideCheckBox" >
         <property name="text" >
          <string>Override CsOptions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="AskFilenameCheckBox" >
         <property name="text" >
          <string>Ask filename every time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="FilePlayFinishedCheckBox" >
         <property name="text" >
          <string>Play file when finished</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>30</y>
        <width>211</width>
        <height>57</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>File type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="FileTypeComboBox" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Sample format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="FileFormatComboBox" />
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>290</y>
       <width>591</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title" >
      <string>Realtime</string>
     </property>
     <widget class="QCheckBox" name="RtOverrideCheckBox" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string>Override CsOptions</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>RT Audio Module</string>
      </property>
     </widget>
     <widget class="QComboBox" name="RtModuleComboBox" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>20</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>280</x>
        <y>20</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Input device (-i)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="RtInputLineEdit" />
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="toolButton_7" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>output device (-o)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="RtOutputLineEdit" />
       </item>
       <item row="1" column="2" >
        <widget class="QToolButton" name="toolButton_8" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Jack client name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="JackNameLineEdit" />
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLineEdit" name="BufferSizeLineEdit" >
     <property name="geometry" >
      <rect>
       <x>320</x>
       <y>10</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="DitherCheckBox" >
     <property name="geometry" >
      <rect>
       <x>390</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>Dither</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Run using:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ExternalRadioButton" >
        <property name="text" >
         <string>External shell</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ApiRadioButton" >
        <property name="text" >
         <string>Csound API</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="AdditionalFlagsCheckBox" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>70</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Additional command line flags</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AdditionalFlagsLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>390</x>
       <y>70</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="BufferSizeCheckBox" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>Buffer Size (-b)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="HwBufferSizeCheckBox" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>40</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>HW Buffer Size (-B)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="HwBufferSizeLineEdit" >
     <property name="geometry" >
      <rect>
       <x>320</x>
       <y>40</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>Environment</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>6</y>
       <width>601</width>
       <height>240</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="1" >
       <widget class="QLineEdit" name="CsdocdirLineEdit" />
      </item>
      <item row="0" column="2" >
       <widget class="QToolButton" name="csdocdirToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="OpcodedirCheckBox" >
        <property name="toolTip" >
         <string>Select the location of the csound opcode plugin  directory (usually set by the installer)</string>
        </property>
        <property name="text" >
         <string>OPCODEDIR</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="OpcodedirLineEdit" />
      </item>
      <item row="1" column="2" >
       <widget class="QToolButton" name="opcodedirToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="SadirCheckBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>SADIR</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="SadirLineEdit" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="sadirToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="SsdirCheckBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>SSDIR</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="SsdirLineEdit" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QToolButton" name="ssdirToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="SfdirCheckBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>SFDIR</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="SfdirLineEdit" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QToolButton" name="sfdirToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QCheckBox" name="IncdirCheckBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>INCDIR</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="IncdirLineEdit" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <widget class="QToolButton" name="incdirToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QCheckBox" name="opcodeXmlDirCheckBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Select the location of the opcode.xml file for syntax highlighting</string>
        </property>
        <property name="text" >
         <string>opcode.xml DIR</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLineEdit" name="opcodeXmlDirLineEdit" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2" >
       <widget class="QToolButton" name="opcodeXmlDirToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="toolTip" >
         <string>Selects the location of the html documentation</string>
        </property>
        <property name="text" >
         <string>Html Doc Directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5" >
    <attribute name="title" >
     <string>External programs</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>20</y>
       <width>581</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>Terminal</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="TerminalLineEdit" />
      </item>
      <item row="0" column="2" >
       <widget class="QToolButton" name="terminalToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Browser</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="BrowserLineEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QToolButton" name="browserToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_16" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Wave Editor</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="WaveEditorLineEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="waveEditorToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_17" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Wave Player</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="WavePlayerLineEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QToolButton" name="wavePlayerToolButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>538</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>619</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpcodedirCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>OpcodedirLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>68</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel" >
     <x>482</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SadirCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SadirLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>68</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel" >
     <x>482</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SfdirCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SfdirLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>68</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel" >
     <x>482</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IncdirCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>IncdirLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>68</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel" >
     <x>482</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SsdirCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SsdirLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>68</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel" >
     <x>482</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BufferSizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>BufferSizeLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>198</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>299</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HwBufferSizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>HwBufferSizeLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>261</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel" >
     <x>298</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AdditionalFlagsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>AdditionalFlagsLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>283</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel" >
     <x>396</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InputFilenameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>InputFilenameLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>123</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel" >
     <x>168</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputFilenameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>OutputFilenameLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel" >
     <x>203</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opcodeXmlDirCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>opcodeXmlDirLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel" >
     <x>163</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
